io/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we do `sdks/java/io` instead of top-level `io`? Basically, every IO is language-specific and part of the SDK of that language. Makes it much easier to manage less top-level things.